test/functional/test_runner.py

```diff
@@ -211,6 +211,7 @@
     'p2p_permissions.py',
     'feature_blocksdir.py',
     'feature_config_args.py',
+    'rpc_getaddressinfo_labels_purpose_deprecation.py',
```

Comment:
I think it's better to just add the test to the `rpc_deprecated.py` test file.

Reply:
> I think it's better to just add the test to the `rpc_deprecated.py` test file.

I started to, then quickly saw that this would completely change that file (here we are not testing a removed RPC but a change in a returned value), and restoring this test file after 0.20 is released would be more complicated than deleting a test file.